### PR TITLE
Fixing the D104 and removing it from the ignore list in Bio/.flake8

### PR DESCRIPTION
**Pull request by azneto**
This pull request addresses issue #1203 

I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ AND the _BSD 3-Clause License_.

I am not happy to be thanked by name in the ``NEWS.rst`` and
``CONTRIB.rst`` files.

I have read the ``CONTRIBUTING.rst`` file and understand that AppVeyor and
TravisCI will be used to confirm the Biopython unit tests and ``flake8`` style
checks pass with these changes.

### CODE REVIEW

File: Bio/Phylo/PAML/__init__.py [hunk 1]
```diff
@@ -1,4 +1,6 @@
-# Copyright (C) 2011 by Brandon Invergo (b.invergo@gmail.com)
-# This code is part of the Biopython distribution and governed by its
-# license. Please see the LICENSE file that should have been included
-# as part of this package.
+"""
+Copyright (C) 2011 by Brandon Invergo (b.invergo@gmail.com)
+This code is part of the Biopython distribution and governed by its
+license. Please see the LICENSE file that should have been included
+as part of this package.
+"""
```
Comment (peterjc) [inline Bio/Phylo/PAML/__init__.py]: That us our standard copyright and license header, which ought to remain as a comment.

**Comment by peterjc** [comment]
I think this is going to need human judgement to write a sensible (short one line) docstring for these files, rather than attempting to autogenerate one from an existing comment. The good news is this is a manageable task.

Note that in many cases the comment was explaining reason for the empty ``__init__.py`` file (I believe this no longer needed as of Python 3.3 which supports implicit namespace packages ). As an aside, it might be worth adding a TODO comment to these files:

```python
# TODO: Remove empty __init__.py once we drop Python 2 support
```

e.g. For ``Bio/GA/Crossover/__init__.py`` something like this would work:

```python
"""Support for crossovers in the Genetic Algorithms module."""
```

**Comment by azneto** [comment]
OK. I'll think I got it.

File: Bio/Statistics/__init__.py [hunk 2]
```diff
@@ -1 +1 @@
-# This is a Python module.
+"""This is a Python module."""
```
Comment (peterjc) [inline Bio/Statistics/__init__.py]: How about:

```python
Basic statistics module.
```

File: Bio/NeuralNetwork/BackPropagation/__init__.py [hunk 3]
```diff
@@ -1 +1,2 @@
-# Allow files in this directory to be imported
+"""Allow files in this directory to be imported"""
```
Comment (peterjc) [inline Bio/NeuralNetwork/BackPropagation/__init__.py]: How about:

```python
"""Back-propagation for neural networks."""
```

File: Bio/NeuralNetwork/Gene/__init__.py [hunk 4]
```diff
@@ -1 +1,2 @@
-# Allow files in this directory to be imported
+"""Allow files in this directory to be imported"""
```
Comment (peterjc) [inline Bio/NeuralNetwork/Gene/__init__.py]: How about:

```python
"""Module for using neural networks to model genetic data."""
```

**Comment by peterjc** [comment]
That's an improvement - I've added comments on the remaining modules in need of a proper one-line description.

**Comment by azneto** [comment]
OK. I've just updated the files.

File: Bio/PDB/mmtf/__init__.py [hunk 5]
```diff
@@ -1,3 +1,4 @@
+"""Support for importing mmtf-python into a Biopython data structure."""
```
Comment (peterjc) [inline Bio/PDB/mmtf/__init__.py]: Perhaps:

```python
"""Support for loading 3D structures stored in MMTF files."""
```

**Comment by peterjc** [comment]
One final suggestion for the MMTK module, and I think this is ready to merge.

**Comment by peterjc** [comment]
Lovely, once the automated testing has finished, I plan to merge this (squashed into one commit).

Again, please let us know or add yourself to the ``NEWS.rst`` and ``CONTRIB.rst`` files as part of a pull request, if you want to be publicly acknowledged.

**Comment by peterjc** [comment]
Thank you.

**Comment by azneto** [comment]
Nice! I'm planning to start working on other issues that involves coding, but I can continue to help with the docstrings as well. Please let me know if you'd like to keep those for newcomers, since they're really basic.

It'd be an honor to integrate the contributors list. (Adhemar Zerlotini https://github.com/azneto)

**Comment by azneto** [comment]
:+1:

**Comment by peterjc** [comment]
Feel free to keep working on the missing docstrings if you like.

**Pull request merged**